tests/foreman/ui/test_repository.py

```diff
@@ -887,23 +887,27 @@ def test_positive_recommended_repos(session, module_entitlement_manifest_org):
     :expectedresults:
 
            1. Shows repositories as per On/Off 'Recommended Repositories'.
-           2. Check last Satellite version Capsule/Tools repos do not exist.
+           2. Check that client repositories are in 'Recommended Repsitories'.
+           2. Check last Satellite version Utils/Capsule/Maintenance repos do not exist.
```

Comment:
Can we make this step more clear? because when I tested PR locally on 6.15, I got recommended repos of `Utils/Capsule/Maintenance` with n-1 version i.e 6.14
```suggestion
           3. Check last Satellite version Utils/Capsule/Maintenance repos do not exist.
```

Reply:
I updated the docstring and addressed the n-1 addition above, I do not believe its necessary